cmd/crawl.go

```diff
@@ -3,23 +3,43 @@ package cmd
 import (
 	"encoding/json"
 	"fmt"
+	"log"
+	"net/url"
+	"strings"
 
 	"github.com/OpenCHAMI/magellan/pkg/crawler"
 	"github.com/spf13/cobra"
 )
 
 var crawlCmd = &cobra.Command{
-	Use:   "crawl",
+	Use:   "crawl [uri] e.g. magellan crawl https://bmc.example.com",
```

Comment:
I think it would be better to put this in a `Long`, since the way this is right now appends `[flags]` at the end of the example command line:
```
# ./magellan crawl --help
Crawl a single BMC for inventory information

Usage:
  magellan crawl [uri] e.g. magellan crawl https://bmc.example.com [flags]
```

If we did something like:

```diff
diff --git a/cmd/crawl.go b/cmd/crawl.go
index 84fb24e..ffc55f1 100644
--- a/cmd/crawl.go
+++ b/cmd/crawl.go
@@ -12,7 +12,11 @@ import (
 )
 
 var crawlCmd = &cobra.Command{
-       Use:   "crawl [uri] e.g. magellan crawl https://bmc.example.com",
+       Use:   "crawl [uri]",
+       Long:  "Crawl a single BMC for inventory information\n" +
+              "\n" +
+              "Example:\n" +
+              "  magellan crawl https://bmc.example.com",
        Short: "Crawl a single BMC for inventory information",
        Args: func(cmd *cobra.Command, args []string) error {
                // Validate that the only argument is a valid URI
```

We would get some nicer output:

```
# ./magellan crawl --help
Crawl a single BMC for inventory information

Example:
  magellan crawl https://bmc.example.com

Usage:
  magellan crawl [uri] [flags]
```

Reply:
done!

Reply:
Looks good!